src/views/TradingCompetition/components/TeamRanks/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Box, Image } from '@pancakeswap-libs/uikit'
+import CakerBunny from '../../pngs/cakers.png'
+import TopTradersCard from './TopTradersCard'
+import Podium from './Podium'
+
+const Wrapper = styled(Flex)`
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    flex-direction: row;
+  }
+`
```

Comment:
If this is the only style you are going for styled systems accept responsive values.

```
<Flex flexDirection={{ xs: 'column', md: 'row' }} />
```

https://styled-system.com/responsive-styles

Reply:
Oh man. This is a game changer, I've been declaring styled comps wherever I've needed to use media queries. This is ace, cheers

Reply:
Had some success with this approach when reassigning margins via props, but not flexDirection. In this instance - I'll stick with the wrapper. Can dig into why this doesn't work later.